### DISCUSSION TITLE
Coral vs Nvidia default models

### DISCUSSION THREAD

**Discussion by Bubbgump209** [blakeblackshear/frigate#7666] [Ask A Question]
It appears the default models for the Coral and Nvidia detectors. One criticism I've seen is the Coral model is "too general" for CCTV use. Is there any benefit to the models that ship with the Nvidia Tensor container? I can run either Coral or Nvidia but admittedly Coral is much easier to maintain.

**Comment by NickM-27** [comment]
All default models currently included in Frigate are trained on the COCO dataset.

**Reply by Bubbgump209** [reply]
So it's all the same source dataset I think I'm understanding.

Follow up question, why are the defaults yolov4-tiny-288,yolov4-tiny-416,yolov7-tiny-416? I don't have a criticism, I'm just trying to understand the reasoning and being a neophyte I would expect yolov7(x?)-640 to be the default as newer versions of YOLO I'd imagine to be faster/more accurate?

**Reply by NickM-27** [reply]
In testing larger models do not perform better, in the next version of frigate 320x320 is going to be the default for nvidia like it is for the coral and openvino models.

**Reply by Bubbgump209** [reply]
Awesome - so a matter of diminishing returns. Excellent. Thanks for the insight and quick responses!
